inst/qml/SEM.qml

```diff
@@ -92,35 +92,49 @@ Form
 				label: qsTr("Factor scaling")
 				values:
 				[
-					{ label: qsTr("Factor loadings")	, value: "factorLoading"	},
-					{ label: qsTr("Factor variance")	, value: "factorVariance"			},
-					{ label: qsTr("Effects coding")		, value: "effectCoding"	},
-					{ label: qsTr("None")				, value: "none"				}
+					{ label: qsTr("Factor loadings"),	value: "factorLoading"	},
+					{ label: qsTr("Factor variance"),	value: "factorVariance"	},
+					{ label: qsTr("Effects coding"),	value: "effectCoding"	},
+					{ label: qsTr("None"),				value: "none"			}
 				]
 			}
 			CheckBox 
-			{ name: "meanStructure"
+			{
+				name: "meanStructure"
 				label: qsTr("Mean structure")
 				checked: eq_intercepts.checked || eq_means.checked || eq_thresholds.checked
-				CheckBox { name: "latentInterceptFixedToZero";		label: qsTr("Latent means fixed to zero"); checked: !eq_means.checked }
-				CheckBox { name: "manifestInterceptFixedToZero";	label: qsTr("Manifest intercepts fixed to zero") 											}
-				CheckBox { name: "manifestMeanFixedToZero";				label: qsTr("Mean of manifest intercepts fixed to zero")							}
+				CheckBox { name: "latentInterceptFixedToZero";		label: qsTr("Latent means fixed to zero");					checked: !eq_means.checked	}
+				CheckBox { name: "manifestInterceptFixedToZero";	label: qsTr("Manifest intercepts fixed to zero")										}
+				CheckBox { name: "manifestMeanFixedToZero";			label: qsTr("Mean of manifest intercepts fixed to zero")								}
 			}
 			
-			CheckBox { name: "orthogonal";						label: qsTr("Assume factors uncorrelated")						}
+			CheckBox { name: "orthogonal";	label: qsTr("Assume factors uncorrelated") }
 		}
 
 		Group
 		{
-
-			CheckBox { name: "exogenousCovariateFixed";					label: qsTr("Exogenous covariate(s) fixed"); 			checked: true	}
+			CheckBox 
+			{
+				name:		"exogenousCovariateFixed"
+				label:		qsTr("Exogenous covariate(s) fixed")
+				id:			fix_x
+				checked:	true
+			}
+			CheckBox 
+			{
+				name:		"exogenousCovariateConditional"
+				label:		qsTr("Condition on exogenous covariate(s)")
+				id:			cond_x
+				// Try to mimic lavaan's default setting for conditional.x
+				checked:	["wls", "dwls", "wlsm", "wlsmv"].includes(estimator.value) && errorCalc.value != "bootstrap"
```

Comment:
this catches most cases, nice. There is a case where the data are ordinal and the estimator is default, then this will not be checked although it should. But that is fine since there is the error message from R showing.